src/wp-includes/user.php

```diff
@@ -81,7 +81,7 @@ function wp_signon( $credentials = array(), $secure_cookie = '' ) {
 	 *
 	 *     @type string $user_login    Username.
 	 *     @type string $user_password Password entered.
-	 *     @type bool   $remember      Whether to 'remember' the user. Increases the time
+	 *     @type bool   $parsed_argsemember      Whether to 'remember' the user. Increases the time
```

Comment:
Replacement of `$r` with `$parsed_args` beats the goal of having "good" variable names if the end result is `$parsed_argsember`

The changes in this file were not needed.

Reply:
I did something really stupid there - Search and Replace `$r` with `$parsed_args`!

I'll work on correcting my error.